pkg/api/orchestrators_test.go

```diff
@@ -298,3 +238,44 @@ func TestDockerceInfoInfo(t *testing.T) {
 	_, e := dockerceInfo(csOrch, false)
 	Expect(e).To(BeNil())
 }
+
+func TestGetKubernetesAvailableUpgradeVersions(t *testing.T) {
```

Comment:
This is one of those "the tests are just as important if not more important than the code" PRs